app/models.py

```
@@ -0,0 +1,22 @@
+
+class ComparisonRequest(BaseModel):
+    repo_path: str
+    old_commit: str
+    new_commit: str
+    repo_name: Optional[str] = None
+
+class ComparisonResponse(BaseModel):
+    status: str
+    message: str
+    diff_file: Optional[str] = None
+    timestamp: datetime = datetime.now()
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)